app/Http/Controllers/GistsController.php

```diff
@@ -62,6 +60,38 @@ public function show($username, $gistId)
 
         return View::make('gistlogs.show')
             ->with('gistlog', $gistlog)
-            ->with('pageTitle', $gistlog->title.' | '.$gistlog->author);
+            ->with('pageTitle', $gistlog->title . ' | ' . $gistlog->author)
+            ->with('isStarredForUser', $gistClient->isStarredForUser($gistId));
+    }
+
+    protected function star(GistClient $client, $gistId)
```

Comment:
Can we move these two methods into a `GistStarController`, and then they'd just be `update` (?) and `delete`... I think?